scout/constants/variant_tags.py

```diff
@@ -29,6 +29,12 @@
     (0.0, 0.016): {"label": "Strong to very strong benign", "color": "success"},
 }
 
+SPLICEAI_SCORE_LABEL_COLOR_MAP = {
+    (0.20, 1.00): {"label": "Predicted impact on splicing", "color": "warning"},
+    (0.10, 0.20): {"label": "Not informative", "color": "secondary"},
+    (0.00, 0.10): {"label": "No impact on splicing", "color": "success"},
+}
+
```

Comment:
Based on this: https://pmc.ncbi.nlm.nih.gov/articles/PMC10357475/table/tbl1/, but suggestions accepted!